tap_spreadsheets_anywhere/__init__.py

```diff
@@ -46,9 +46,9 @@ def discover(config):
         try:
             modified_since = dateutil.parser.parse(table_spec['start_date'])
             target_files = file_utils.get_matching_objects(table_spec, modified_since)
-            sample_rate = table_spec.get('sample_rate',10)
+            sample_rate = table_spec.get('sample_rate',5)
             max_sampling_read = table_spec.get('max_sampling_read', 1000)
-            max_sampled_files = table_spec.get('max_sampled_files', 5)
+            max_sampled_files = table_spec.get('max_sampled_files', 50)
```

Comment:
The existing defaults are arbitrary - so I'm not too worried about changing them. Just want to understand the motivation.

Reply:
The defaults were only changed because our client's data. Essentially the files we were pulling had columns that seldom had data (i.e. lots of null values), which wasn't an accurate representation of schema.

Reply:
I meant to not commit the changes, but they apparently slipped through.